### PR TITLE
Unit Tests for Comments

### PR DESCRIPTION
**Pull request by le-sun**
Added unit tests
Fixed comment patch bug + other minor bug fixes

### CODE REVIEW

File: src/researchhub_comment/related_models/rh_comment_model.py [hunk 1]
```diff
@@ -136,7 +136,7 @@ def users_to_notify(self):
 
     def update_comment_content(self):
         celery_create_comment_content_src.apply_async(
-            (self.id, self.comment_content_json), delay=5
+            (self.id, self.comment_content_json), countdown=2
```
Comment (calvinhlee23) [inline src/researchhub_comment/related_models/rh_comment_model.py:139]: can you explain what countdown does?

**Review by yattias — APPROVED** [review]
LGTM

File: src/researchhub_comment/views/rh_comment_view.py [hunk 2]
```diff
@@ -74,7 +73,7 @@ def censor_comment(comment):
     """
 
     for bounty in comment.bounties.iterator():
-        cancelled = bounty.cancel()
+        cancelled = bounty.close()
```
[the remark this answers is not in the document]
Comment (le-sun) [reply src/researchhub_comment/views/rh_comment_view.py:76]: Yep, this fixes the bug that Jeff had

[the remark this answers is not in the document]
Comment (le-sun) [reply src/researchhub_comment/views/rh_comment_view.py:76]: It was misnamed. Always meant to be `.close()`

File: src/researchhub_comment/related_models/rh_comment_model.py (hunk 1 above)
Comment (le-sun) [reply src/researchhub_comment/related_models/rh_comment_model.py:139]: The amount of time before Celery can execute the task

**Pull request merged**